src/tools/auth0/handlers/emailProvider.ts

```diff
@@ -30,34 +31,38 @@ export default class EmailProviderHandler extends DefaultHandler {
   async processChanges(assets: Assets): Promise<void> {
     const { emailProvider } = assets;
 
-    // Do nothing if not set
     if (!emailProvider) return;
 
-    if (Object.keys(emailProvider).length > 0) {
```

Comment:
This if statement is instead replaced by a negated condition that early returns.